IFC4x3/Properties/d/DiscontinuitySetID_1OpK8YZrP9tRVsNinKvGX7/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="DiscontinuitySetID_1OpK8YZrP9tRVsNinKvGX7" Name="DiscontinuitySetID" UniqueId="58cd4222-8f56-49dd-b7f6-5ecc54e50847" PrimaryDataType="IfcLabel" />
```

Comment:
Should `IfcRoot`.`GlobalId` or .`Name` be used here? Maybe `IfcElement`.`Tag`?

Reply:
I don't think that this id is a unique id for this a particular instance of a discontinuity but more the identity of a set (a reference). This could mean that this should actually be a "Product Assignment" usage?
Sorry @j-weil. Need you again.

Reply:
Removed. Instead use aggregation for the set and use Name for the composition id.